Examples/Algorithms/Geant4/include/ActsExamples/Geant4/SensitiveSurfaceMapper.hpp

```diff
@@ -62,10 +67,12 @@ class SensitiveSurfaceMapper {
   /// hierarchy and applies name remapping to the Physical volumes
   /// of the Geant4 geometry.
   ///
+  /// @param gctx the geometry context
   /// @param g4PhysicalVolume the current physical volume in process
   /// @param motherPosition the absolute position of the mother
   /// @param sCounter  a counter of how many volumes have been remapped
-  void remapSensitiveNames(G4VPhysicalVolume* g4PhysicalVolume,
+  void remapSensitiveNames(const Acts::GeometryContext& gctx,
```

Comment:
why do we need the geometry context here? does that make sense to pass it around during simulation?